jwst/flatfield/flat_field.py

```diff
@@ -405,8 +397,21 @@ def do_NIRSpec_flat_field(output_model,
         interpolated_flats = None
 
     any_updated = False
+    if exposure_type == "NRS_MSASPEC":
+        slits = nirspec.get_open_slits(output_model)
     for (k, slit) in enumerate(output_model.slits):
         log.debug("Processing slit %s", slit.name)
```

Comment:
Most of the examples in the documentation for logger, even for Python 3, use the old % syntax with a comma-separated list of arguments to the log.debug function (and log.info, etc).  The doc says this is for backward compatibility, but I read somewhere that this style is used because it allows logger to use any formatter.  Note that this isn't the same as this kind of formatting:  "abc%s" % "def"; instead, it's %s (%d, etc) in the string, but the values are passed as arguments.